### PR TITLE
Feature/149133521 cms menu access

### PR DESCRIPTION
**Pull request by RianaFerreira**
https://www.pivotaltracker.com/story/show/149133521

Only allow Admin users to access the CMS menu link.
Prevent direct navigation to CMS pages with the url path for unauthenticated users.

### CODE REVIEW

File: spec/controllers/admin/cms/collections_controller_spec.rb [hunk 1]
```diff
@@ -12,6 +12,13 @@
     )
   end
 
+  before do
```
Comment (GlenCrawford) [inline spec/controllers/admin/cms/collections_controller_spec.rb]: These three are all the same, right? Is this app using Devise, if so, there are some test helper methods that they provide (described in the README) that allow you to just do something like:

```ruby
let(:user) { User.create! ...create an admin user... }
before { sign_in user }
```

**Comment by GlenCrawford** [comment]
Oh, also, isn't this PR meant to include a new CMS link in the menu? Did that change not get added to the PR, maybes?

**Comment by RianaFerreira** [comment]
The html changes for adding the link to the menu have been handled on this PR:
https://github.com/slnswgithub/nsw-state-library-amplify/pull/12

**Comment by RianaFerreira** [comment]
Accidentally closed the PR when adding the last comment.

**Pull request merged**